src/VolleyManagement.Backend/VolleyManagement.Services/Authorization/AuthorizationService.cs

```diff
@@ -63,11 +53,11 @@ public void CheckAccess(AuthOperation operation)
         /// </summary>
         /// <param name="requestedOperations">Operations to check</param>
         /// <returns>An instance of <see cref="AllowedOperations"/> class</returns>
-        public AllowedOperations GetAllowedOperations(List<AuthOperation> requestedOperations)
+        public AllowedOperations GetAllowedOperations(ICollection<AuthOperation> requestedOperations)
         {
             if (requestedOperations == null)
             {
-                throw new ArgumentNullException("Requested operations list shouldn't be null!");
+                throw new ArgumentNullException($"Requested operations list shouldn't be null!");
```

Comment:
?

Reply:
it is the best way, warning "Can not resolve symbol "Requested operations ...""

Reply:
What parameter this overload accepts?

Reply:
it accepts string but sonar warns like "Can not resolve symbol"

Reply:
![image](https://user-images.githubusercontent.com/17709640/36744446-88f2bca4-1bf5-11e8-8b6b-0069fc4bbe28.png)

Reply:
![image](https://user-images.githubusercontent.com/17709640/36744466-98d2ccb8-1bf5-11e8-9372-81c78886055c.png)

Reply:
Do you understand why it reports such error?

Reply:
no meter

Reply:
I don't understand

Reply:
ok, returned to previous state

Reply:
returned!

Reply:
**ArgumentNullException Constructor (String)**
Initializes a new instance of the ArgumentNullException class with the name of the parameter that causes this exception.
**ArgumentNullException Constructor (String, String)**
Initializes an instance of the ArgumentNullException class with a specified error message and the name of the parameter that causes this exception.

Maybe to solve this, is better to use ArgumentNullException Constructor (String, String)?